.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
I suggest you use the .gitignore file to make git ignore specific files and folders like the .vscode folder. You can see how to do it [here](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/).